basesuite/basesuite.go

```diff
@@ -0,0 +1,134 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build !windows
+
+package basesuite
+
+import (
+	"fmt"
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"log"
+	"path/filepath"
+	"time"
+)
+
+const (
+	configOutputPath     = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+	agentConfigDirectory = "agent_configs"
+	extraConfigDirectory = "extra_configs"
+	MinimumAgentRuntime  = 3 * time.Minute
+)
+
+type ITestRunner interface {
```

Comment:
Is this [different from the ITestRunner interface](https://github.com/aws/amazon-cloudwatch-agent-test/blob/011e3ae8fb3789d620f245295536b171b73a2e37/test/test_runner/base_test_runner.go)?

Reply:
ah crap I didn't delete my copy paste